src/game/server/swarm/asw_marine.h

```diff
@@ -285,6 +285,8 @@ class CASW_Marine : public CASW_VPhysics_NPC, public IASWPlayerAnimStateHelpers,
 	virtual bool ShouldNotDistanceCull();	// BenLubar(deathmatch-improvements):
 	virtual Disposition_t IRelationType( CBaseEntity *pTarget );	// BenLubar(deathmatch-improvements):
 
+	CNetworkVar(Vector, m_vecCustLaserColor);
```

Comment:
```suggestion
	CNetworkVector(m_vecCustLaserColor);
```

also consider moving the variable to the marine resource